src/stores/utils/web3.js

```diff
@@ -1,13 +1,29 @@
 import Web3 from 'web3'
 import Web3Utils from 'web3-utils'
 
+const updateTitle = (networkName = 'No chain specified') => {
+  const titleReplaceString = '%c'
+  let appTitle = process.env.REACT_APP_TITLE || null
```

Comment:
May we have some default title if no title mentioned in the `.env` file? My suggestion is `TokenBridge UI app`

Reply:
That's shown by default in index.html (it's hardcoded), and the function just returns if there's no REACT_APP_TITLE.

Is it necessary to do something else?

Reply:
We need to change it as so it is managed from one place. If we decide to change default value it seems to be logical to change it in `updateTitle` since it is responsible for the title generation rather than the index.html. @patitonar what do you think?

Reply:
Sure, makes sense.

Now the default title should be always `TokenBridge UI app`, no matter what the index.html file says. That's also the case if it is absent from .env